nservicebus/pipeline/unit-of-work.md

```diff
@@ -6,6 +6,43 @@ redirects:
 - nservicebus/unit-of-work-in-nservicebus
 ---
 
+## Using a transaction scope
+
+If your business transaction is spread across multiple handlers there is always a risk of partial updates since one handler might succeed in updating the data while other won't. To avoid this use a unit of work that wraps all handlers in a `TransactionScope` and makes sure that there are no partial updates. Use following code to enable a wrapping scope:
+
+snippet:UnitOfWorkWrapHandlersInATransactionScope
+
+NOTE: This requires the selected storage to support enlisting in transaction scopes.
```

Comment:
I think this particular sentence refers to the storage in which the user's business data is stored.

I have a vague memory of @andreasohlund saying that writes to NSB storage might (with the exclusion of Outbox) happen outside of this `TransactionScope`.

Of course, there's also fat chance I might be wrong here, so experts should feel free to correct me.

From doco perspective, should we be explicit about what storage is meant here?